provers/groth16/src/qap.rs

```diff
@@ -0,0 +1,96 @@
+use crate::common::*;
+use lambdaworks_math::{fft::polynomial::FFTPoly, polynomial::Polynomial};
+
+#[derive(Debug)]
+pub struct QAP {
```

Comment:
Please prefer declarative names over acronyms
```suggestion
pub struct QuadraticArithmeticProgram {
```

Reply:
done